boost.yaml

```diff
@@ -0,0 +1,80 @@
+name: boost
```

Comment:
Absolutely honest question – how come there are ca. 40 applications bundling boost but only yours requires so many patches?

Reply:
> Absolutely honest question – how come there are ca. 40 applications bundling boost but only yours requires so many patches?

I know that you are against patches that don't seem absolutely necessary to you. And because of that, we got stuck with the broken KDE runtime. All current KDE runtimes (based on Freedesktop 1.6) are broken due to the bug in glibc. This causes instability of applications based on PyQt5/QtWebEngine.
https://github.com/flatpak/freedesktop-sdk-images/issues/97#issuecomment-383414574
https://github.com/flatpak/freedesktop-sdk-images/issues/97#issuecomment-392346027
And this will not change until the release of KDE 5.12 (based on Freedesktop 18.08).
https://blogs.gnome.org/alexl/2018/10/11/moving-away-from-the-1-6-freedesktop-runtime/#comment-2252
See also:
https://sourceware.org/git/?p=glibc.git;a=blob;f=elf/dl-close.c;hb=9e78f6f6e7134a5f299cc8de77370218f8019237#l809
https://sourceware.org/git/?p=glibc.git;a=blob;f=elf/dl-close.c;h=ecd6729704ea3294a690d55ac2469f80087f65e9;hb=HEAD#l819

Reply:
> I ask about your boost module specifically. I don't need to hear anything about KDE runtime or glibc.

For each patch you have a short description and/or link to the bugzilla issue. What exactly do you not understand?

If you ask why these changes didn't go upstream, here you will find the answer:
> In Fedora we have kept lugging the patch along for years, it would be really nice to be able to retire it.

https://svn.boost.org/trac10/ticket/6701#comment:9

Reply:
I think my question was clear. What exactly these patches fix in Widelands?

Reply:
> I think my question was clear. What exactly these patches fix in Widelands?

These patches are for Boost, not for Widelands itself. I'm just trying to provide good quality modules that can be reused in other packages (with minimal changes).
What kind of question is that? It's as if you were asking how a particular patch in glibc, Gtk+ or Mesa affects XYZ app.